config/locales/zh.yml

```diff
@@ -0,0 +1,728 @@
+      email_notifications_question: '如何修改电子邮件通知？'
+      email_notifications_answer: '转到<strong>％{帐户} </ strong>页面。您可以在此处修改电子邮件，密码，个人资料图片和个人简介。您也可以删除您的帐户。删除帐户后，所有数据都会被删除。'
+    home:
+      update_pwd_modal:
+        title: '更新密码'
+        body: '我们已为用户实施了更好的安全措施，请％{update_password_link}'
+        update_link: '更新您的密码。'
+      stories: 故事
+      not_signed_in:
+        main_message_one: '与亲人更好的沟通可以改善心理健康。'
+        message_text_one: '我们需要彼此的支持来消除污名。'
+        message_one: '与亲人分享您的故事并获得应有的支持'
+        message_two: '是您，没有附带条件，并具有安全性和隐私性'
+        message_three: '记录您的情绪，药物和自我保健'
+        main_message_two: '我们是开源的，因为心理健康应该是开放的。'
+        message_text_two: '我们<strong>始终</ strong>一直在寻找您的％{help}和％{feedback}！'
+        help: 救命
+      signed_in_empty:
+        main_message_one: '％{name}，您好！'
+        message_text_one: '这里是您如何开始分享故事的方法。'
+        main_message_two: '向我们介绍你自己。'
+        cell_one: '自定义您的％{profile}-添加个人简介和个人资料图片<br>添加自定义％{moods}以最好地描述您的情绪<br>添加％{allies}-您想与之分享的朋友，家人和其他人'
+        main_message_three: '写你的故事。'
+        cell_two: '创建自定义％{categories}以帮助组织您的时刻<br>写出％{moments}-影响您心理健康的事件和情况'
+        main_message_four: '好好照顾自己。'
+        cell_three: '％{strategize}您的自我保健计划！<br>目前正在服用％{medications}？跟踪它并创建提醒！<br>需要额外的支持吗？加入或创建％{group}以打开'
+        main_message_five: '有任何疑问或疑虑吗？'
+        message_text_three: '请查看我们的％{faq}页面或％{email_link}！'
+        strategize: 制定策略
+    partners:
+      description: '这些是令人难以置信的组织，我们受到启发并有幸与之合作。我们很乐意与您的组织合作，通过％{email}向我们发送电子邮件。'
+    privacy:
+      summary: 摘要
+      summary_description: '<p>本隐私政策适用于本网站，并控制选择使用本网站的用户的隐私。</ p> <p>鉴于网站的敏感内容，隐私是我们极为关注的重要问题。该政策旨在概述用户数据的收集方式，用途以及保护方式。</ p>'
+      user_data: '用户数据'
+      user_data_description: '<p>用户旨在为我们提供他们一生的个人摘录（在“瞬间”功能中）和药物轶事。来自Moments功能的数据将与用户选择与之共享的任何人共享。药物功能的数据仅对用户可见。这两个数据都不会用于任何其他目的，也不会交还给外部各方。</ p> <p>该应用程序还包含有关支持团体和应对心理健康挑战的策略的数据。只有拥有查看权限的用户才能访问此数据。来自两个功能部件的数据可以公开显示，以向当前和将来的用户展示。但是，用户身份将被保密。</ p>'
+      cookies: 'Cookies的使用'
+      cookies_description: '<p>该网站使用cookie来改善用户访问网站时的体验。在适用的情况下，本网站使用cookie控制系统，允许用户首次访问该网站时允许或禁止在其计算机/设备上使用cookie。</ p> <p> Cookie是保存给用户的小文件''的计算机硬盘驱动器，用于跟踪，保存和存储有关用户互动和网站使用情况的信息。这使网站可以通过其服务器为用户提供该网站内的量身定制的体验。</ p> <p>建议用户，如果他们希望拒绝使用和保存该网站上的cookie并将其保存到计算机硬盘上他们应在其网络浏览器的安全设置中采取必要的步骤，以阻止该网站及其外部服务供应商的所有cookie。</ p> <p>该网站使用跟踪软件来监视访问者，以更好地了解他们的使用方式。该软件由Google Analytics（分析）提供，它使用Cookie来跟踪访问者的使用情况。该软件会将Cookie保存到您计算机的硬盘中，以跟踪和监视您对网站的参与和使用情况，但不会存储，保存或收集个人信息。您可以阅读Google的％{google_privacy_policy}以了解更多信息。</ p>'
+      social_media: '社交媒体平台'
+      social_media_description: '<p>通过本网站参与的外部社交媒体平台进行的沟通，参与和采取的措施分别根据条款和条件以及每个社交媒体平台所持有的隐私政策而定制。</ p> <p>建议用户明智地使用社交媒体平台，并就其自身的隐私和个人详细信息谨慎/谨慎地与他们进行交流/互动。该网站绝不会通过社交媒体平台询问个人或敏感信息，并鼓励希望讨论敏感细节的用户通过电话或电子邮件等主要交流渠道与他们联系。</ p> <p>该网站可能会使用社交共享按钮这有助于直接从网页到相关的社交媒体平台共享Web内容。建议用户在使用此类社交共享按钮之前自行决定这样做，并请注意，社交媒体平台可以分别通过您的社交媒体平台帐户跟踪和保存您共享网页的请求。</ p>'
+      contact: '联系我们'
+      contact_description: '<p>如果有任何疑问或疑问，请％{email_link}。</ p> <p class =“ noMarginBottom”> <em>根据％{jamie_king_media}改编的隐私权政策。</ em> </ p>'
+    resources:
```

Comment:
There were changes made to this section of the translations! You'll want to rebase from master!